plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -1327,11 +1315,21 @@ public void testReadCdfChanges()
                         """);
     }
 
-    @Test
-    public void testReadCdfChangesOnPartitionedTable()
+    @DataProvider
+    public Object[][] mappingModesProvider()
```

Comment:
I would move to the bottom in the class for locating the method after the usages. Same for TestDeltaLakeChangeDataFeedCompatibility. 
```suggestion
    public Object[][] columnMappingModeDataProvider()
```